migration/1676472760533-FillProjectDescriptionSummaryProjectUpdateContentSummary.ts

```diff
@@ -0,0 +1,59 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { Project, ProjectUpdate } from '../src/entities/project';
+import { getHtmlTextSummary } from '../src/utils/utils';
+
+export class FillProjectDescriptionSummaryProjectUpdateContentSummary1676472760533
+  implements MigrationInterface
+{
+  async up(queryRunner: QueryRunner): Promise<void> {
+    let skip = 0;
+    while (true) {
+      const [projects, count] = await queryRunner.manager.findAndCount<Project>(
+        Project,
+        {
+          select: ['id', 'description'],
+          order: { id: 'ASC' },
+          skip,
+        },
+      );
+
+      await Promise.all(
+        projects.map(project => {
+          queryRunner.manager.update(
+            Project,
+            { id: project.id },
+            { descriptionSummary: getHtmlTextSummary(project.description) },
+          );
+        }),
+      );
+
+      skip += projects.length;
+      if (skip >= count) break;
+    }
+
+    skip = 0;
+    while (true) {
+      const [projectUpdates, count] =
+        await queryRunner.manager.findAndCount<ProjectUpdate>(ProjectUpdate, {
```

Comment:
I think you should specify a take amount, this will fetch all the records in the db might be an expensive operation and block the migration.

Reply:
The docs don't specify a ```take``` by default in the resulting query so I assume it fetches all.